metaflow/graph.py

```diff
@@ -63,6 +65,7 @@ def __init__(self, func_ast, decos, doc):
         self.in_funcs = set()
         self.split_parents = []
         self.matching_join = None
+        # self.is_multinode_step = any(isinstance(deco, MultinodeDecorator) for deco in decos)
```

Comment:
can we enable `is_multinode_step`? It seems like a useful piece of info

Reply:
Oops, sorry I have left this commented (should have removed it). It was in the earlier version where multinode decorator had num_nodes attribute. But indeed when I add the proper linting, probably will need this.